_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -107,7 +107,7 @@ The `video` element automatically plays some audio for more than 3 seconds.
 The `video` element is `muted`.
```

Comment:
This test case appears in other rule i.e. 80f0bf but the description is slightly different here. For example description in inapplicable example 1 of ACT-R 80f0bf is 'The `<video>` element audio autoplays for longer than 3 seconds, but is `muted`.'